colossalai/utils/profiler/__init__.py

```diff
@@ -1 +1,2 @@
-from .comm_profiler import enable_communication_prof, communication_prof_show
+# from .comm_profiler import enable_communication_prof, communication_prof_show
+from .common import ProfilerActivity, colossalai_profiler

```

Comment:
Will this change cause the test_profiler unit testing to fail?

Reply:
Definitely. Need to update unitests after the update of specific profilers.

Reply:
Sure. Skip it.